app/upgrades/v1/upgrades.go

```diff
@@ -60,16 +64,43 @@ func CreateUpgradeHandler(upgradeDataStr UpgradeDataString) upgrades.UpgradeHand
 				return nil, err
 			}
 
+			// Re-initialise the mint module as we have replaced Cosmos SDK's
+			// mint module with our own one.
+			// TODO: properly initialise the mint module in the upgrade handler
```

Comment:
if we are adding new module and deleting new modues, then `StoreUpgarades` part of the upgrade must be updated i.e the part that now contains:
```
		StoreUpgrades: store.StoreUpgrades{
			Deleted: []string{ZoneConciergeStoreKey},
		},
```

Reply:
We are not adding new module. It's more like that we replace Cosmos SDK's `x/mint` with our own `x/mint`, with the same DB schema and DB key space. That's how the software upgrade passed

Reply:
So if I understand correctly, whatever was stored under `cosmos/x/mint` module keys, will be overwritten by the new values from `babylon/x/mint` but still be stored under old keys ?

Reply:
Yes. And note that the key prefix for this module is same as Cosmos SDK x/mint, i.e., it is `"mint"`.